sql/core/src/main/scala/org/apache/spark/sql/streaming/ui/StreamingQueryStatusListener.scala

```diff
@@ -115,7 +115,7 @@ private[sql] class StreamingQueryStatusListener(
   }
 }
 
-private[sql] class StreamingQueryData(
+private[spark] class StreamingQueryData(
```

Comment:
[15c0cf9](https://github.com/apache/spark/pull/39233/commits/15c0cf996ce4b0eff7d0168915fb9a9ccb6938b6) change `private[sql]` to `private[spark]` and move `StreamingQueryDataSerializer` to package `org.apache.spark.status.protobuf.sql`